cuBERT_topic_modelling/ctfidf.py

```diff
@@ -0,0 +1,65 @@
+from cuml.feature_extraction._tfidf import TfidfTransformer
+import cupyx.scipy.sparse as sp
+import cupy as cp
+from cuml.common.sparsefuncs import csr_row_normalize_l1
+
+
+class ClassTFIDF(TfidfTransformer):
+    """
+    A Class-based TF-IDF procedure using scikit-learns
+    TfidfTransformer as a base. C-TF-IDF can best be
+    explained as a TF-IDF formula adopted for multiple classes
+    by joining all documents per class. Thus, each class
+    is converted to a single document instead of set of documents.
+    Then, the frequency of words **t** are extracted for
+    each class **i** and divided by the total number of
+    words **w**. Next, the total, unjoined, number of documents
+    across all classes **m** is divided by the total sum of
+    word **i** across all classes.
+    """
+
+    def __init__(self, *args, **kwargs):
+        super(ClassTFIDF, self).__init__(*args, **kwargs)
+
+    def fit(self,
+            X: sp.csr_matrix,
```

Comment:
X is a cupy gpu matrix no ?

Reply:
`sp` is actually `import cupyx.scipy.sparse as sp`, I think I shuld change the alias here